modules/nw-ovn-k-anp-best-practices.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included:
+// networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-recommended-practices.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="anp-best-practices_{context}"]
+= Designing AdminNetworkPolicy
+
+When building `AdminNetworkPolicy` (ANP) resources, you should consider the following when creating your policies:
+
+* You should create `AdminNetworkPolicy` at different priorities so that precedence is well defined. It is not guaranteed which policy will take precedence when overlapping ANP are created.
+
+* Administrators should build ANP that apply to user namespaces not system namespaces.
+
+[IMPORTANT]
+====
+Applying ANP and BANP to system namespaces (`default`, `kube-system`, any namespace whose name starts with `openshift-`, etc) is not supported, and this can leave your cluster unresponsive and in a non-functional state.
+====
+
+* Since `0-100` is the supported priority range, it is always better when designing your ANP to use a middle range like `30-70` because this leaves some placeholder
+for priorities before and after. Even in the middle range, you might want to leave gaps so that as your infrastructure requirements evolve over time, you are able to insert new ANPs when needed at the right priority level. If you pack your ANPs, then you might need to recreate all of them to accommodate any changes in the future.
+
+* When using `0.0.0.0/0` or `::/0` to create a strong `Deny` policy, it is recommended that you check to make sure that you have higher priority `Allow` or `Pass` rules for essential traffic.
+
+* Use `Allow` when you want to ensure that a connection is allowed no matter what. An `Allow` rule in an ANP means that the connection will always be allowed, and `NetworkPolicy` will be ignored.
+
+* Use `Pass` if you want to "not deny" the connection, and allow developers to decide for themselves whether to allow or deny the connection according to their own `NetworkPolicy` objects.`Pass` delegates policy decision to the `NetworkPolicy` layer.
+
+* Selectors across multiple rules should not overlap. This ensures that that the same IPs do not appear in multiple policies, which can cause performance and scale limitations.
```

Comment:
```suggestion
* Ensure that the selectors across multiple rules do not overlap so that the same IPs do not appear in multiple policies, which can cause performance and scale limitations.
```